### ISSUE TITLE
Upgrade Finder optional item source

### ISSUE THREAD

**Issue by EvilBlood** [Voulk/QuestionablyEpic#1011]
For M+ players it would be cool if you could use the upgrade finder without having to select a raid. My people and I only play M+ and we don't need or use raid items. but we always get some suggested in the upgrade finder because we can't deselect them. This makes the upgrade overview especially with newer characters very confusing because you first have to filter out what is interesting for you. Maybe you can add a checkbox for the individual sources or a drop down menu where you can select the item sources.

**Comment by Voulk** [comment]
Yeah I might add a "None" option for raid so that you can effectively ignore that source. I'll see if I can have it in by Season 3.
